.build/microsoft-azure/build-vm.sh

```diff
@@ -0,0 +1,156 @@
+export runOutputName=nextcloudAioImg01ro
+
+# create resource group
+# az group create -n $imageResourceGroup -l $location
+
+
+#
+# Create User-Assigned Managed Identity and Grant Permissions
+#
+
+# create user assigned identity for image builder to access the storage account where the script is located
+export identityName=nextcloud-aio-$(date +'%s')
+az identity create -g $imageResourceGroup -n $identityName
+
+# get identity id
+export imgBuilderCliId=$(az identity show -g $imageResourceGroup -n $identityName | grep "clientId" | cut -c16- | tr -d '",')
+
+# get the user identity URI, needed for the template
+export imgBuilderId=/subscriptions/$subscriptionID/resourcegroups/$imageResourceGroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/$identityName
+
+# download preconfigured role definition example
+rm -f aibRoleImageCreation.json
+curl https://raw.githubusercontent.com/Azure/azvmimagebuilder/main/solutions/12_Creating_AIB_Security_Roles/aibRoleImageCreation.json -o aibRoleImageCreation.json
+
+export imageRoleDefName="Azure Image Builder Image Def"$(date +'%s')
+
+# update the definition
+sed -i -e "s/<subscriptionID>/$subscriptionID/g" aibRoleImageCreation.json
+sed -i -e "s/<rgName>/$imageResourceGroup/g" aibRoleImageCreation.json
+sed -i -e "s/Azure Image Builder Service Image Creation Role/$imageRoleDefName/g" aibRoleImageCreation.json
+
+# create role definitions
+az role definition create --role-definition ./aibRoleImageCreation.json
+
+# Wait until it is created (no check possible)
+sleep 1m
+
+# grant role definition to the user assigned identity
+az role assignment create \
+    --assignee $imgBuilderCliId \
+    --role "$imageRoleDefName" \
+    --scope /subscriptions/$subscriptionID/resourceGroups/$imageResourceGroup
+
+#------------------------------------------------------------
+
+#
+# 2. Modify HelloImage Example
+#
+
+# download the example and configure it with your vars
+sed -i -e "s/<subscriptionID>/$subscriptionID/g" image-build.json
+sed -i -e "s/<rgName>/$imageResourceGroup/g" image-build.json
+sed -i -e "s/<region>/$location/g" image-build.json
+sed -i -e "s/<imageName>/$imageName/g" image-build.json
+sed -i -e "s/<runOutputName>/$runOutputName/g" image-build.json
+
+sed -i -e "s%<imgBuilderId>%$imgBuilderId%g" image-build.json
+
+#------------------------------------------------------------
+
+#
+# 3. Create the image
+#
+
+# submit the image confiuration to the VM Image Builder Service
+
+az resource create \
+    --resource-group $imageResourceGroup \
+    --properties @image-build.json \
+    --is-full-object \
+    --resource-type Microsoft.VirtualMachineImages/imageTemplates \
+    -n image-build01
+
+# wait approx 1-3mins, depending on external links
+
+# start the image build
+
+az resource invoke-action \
+     --resource-group $imageResourceGroup \
+     --resource-type  Microsoft.VirtualMachineImages/imageTemplates \
+     -n image-build01 \
+     --action Run 
```

Comment:
fails here currently with 
`Deployment failed. Correlation ID: f50b1b88-ad5d-46f7-97a9-f0c5dc88f9ce. During the image build a failure has occurred. Please review the build log to identify which build/customization step failed. For more troubleshooting steps go to https://aka.ms/azvmimagebuilderts. Image build log location: https://5m9nmz8iiagrfdmsuf3d6kwj.blob.core.windows.net/packerlogs/a6bb0707-3f8c-4f63-ac14-371f4a6ef8a9/customization.log. OperationId: 94e69e80-0265-4a68-a717-741aeb301203. Use this operationId to search packer logs.`

Reply:
fixed